src/SFA.DAS.Commitments.Api.IntegrationTests/ApiHost/IntegrationTestServer.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
```

Comment:
could you split down the classes to their own files?

Reply:
When I was coding, I was in two minds whether to split them out or keep them together as they work in concert, but it is cleaner to have them in their own files, so I've split them out.